`pytest` integration tests fail on my machine

`pytest` integration tests fail on my machine due to the predicted paths not existing; this seems to be due to that in some cases `platform.processor` returns an [empty string](https://docs.python.org/3/library/platform.html#platform.processor). I haven't looked into the why for this yet. However `platform.processor` returning an empty string deviates from the Go code which *does* find the architecture and hence the filenames differ.

```
user@muja ~ € python3 -q
>>> import platform
>>> platform.processor()
''
```

```
user@muja image-builder-cli € sudo pytest
=================================================================== test session starts ====================================================================
platform linux -- Python 3.13.1, pytest-8.3.4, pluggy-1.5.0
rootdir: /home/user/src/redhat/image-builder-cli
collected 3 items                                                                                                                                          

test/test_container.py ..F                                                                                                                           [100%]

========================================================================= FAILURES =========================================================================
__________________________________________________________ test_container_manifest_generates_sbom __________________________________________________________
tmp_path = PosixPath('/tmp/pytest-of-root/pytest-4/test_container_manifest_genera0'), build_container = 'image-builder-test'

    @pytest.mark.skipif(os.getuid() != 0, reason="needs root")
    def test_container_manifest_generates_sbom(tmp_path, build_container):
        output_dir = tmp_path / "output"
        output_dir.mkdir()
        subprocess.check_call([
            "podman", "run",
            "--privileged",
            "-v", f"{output_dir}:/output",
            build_container,
            "manifest",
            "minimal-raw",
            "--distro", "centos-9",
            "--with-sbom",
        ], stdout=subprocess.DEVNULL)
        arch = platform.processor()
        fn = f"centos-9-minimal-raw-{arch}/centos-9-minimal-raw-{arch}.image-os.spdx.json"
        image_sbom_json_path = output_dir / fn
>       assert image_sbom_json_path.exists()
E       AssertionError: assert False
E        +  where False = exists()
E        +    where exists = PosixPath('/tmp/pytest-of-root/pytest-4/test_container_manifest_genera0/output/centos-9-minimal-raw-/centos-9-minimal-raw-.image-os.spdx.json').exists
```

--- a/test/test_container.py
+++ b/test/test_container.py
@@ -42,7 +42,7 @@ def test_container_manifest_generates_sbom(tmp_path, build_container):
         "--distro", "centos-9",
         "--with-sbom",
     ], stdout=subprocess.DEVNULL)
-    arch = platform.processor()
+    arch = platform.machine()
     fn = f"centos-9-minimal-raw-{arch}/centos-9-minimal-raw-{arch}.image-os.spdx.json"
     image_sbom_json_path = output_dir / fn
     assert image_sbom_json_path.exists()

